app/features/send/send-confirmation.tsx

```diff
@@ -213,7 +203,7 @@ export const PayBolt11Confirmation = ({
 
   return (
     <BaseConfirmation
-      amount={inputAmount}
+      amount={data?.totalAmountToSend ?? inputAmount}
```

Comment:
this makes sure that the confirmation page shows actual amount that will be spent (amount returned by the mint) as soon as we have it

Reply:
yeah that would be better than current approach

Reply:
added skeleton here 173e0fc6626720609436ad22cf9b016e2c836ce7